assets/src/edit-story/components/header/buttons/publish.js

```diff
@@ -39,6 +40,8 @@ import { usePrepublishChecklist } from '../../inspector/prepublish';
 import { PRE_PUBLISH_MESSAGE_TYPES } from '../../../app/prepublish';
 import { ButtonContent, WarningIcon } from './styles';
 
+export const ONE_MINUTE_IN_MS = 60 * 1000;
```

Comment:
Should this live here https://github.com/google/web-stories-wp/blob/main/assets/src/edit-story/constants.js

Reply:
As long is as it's only used by this one component it's probably easier to just keep here so it's not accidentally forgotten with a refactoring.